### ISSUE TITLE
Implement Email, Notifications and Elasticsearch

### ISSUE THREAD

**Issue by niranjan94** [fossasia/open-event-orga-server#4196]
- Emails & notifications (High priority)
	- [x] Session submission - to organizer
	- [x] Session approval/rejection - to speakers
	- [x] Session acceptance/rejectance - to organizer
 	- [x] Event invoice - monthly - to organizer
    - [x] Event invoice - Follow up email if not paid -  to organizer
 	- [x] Ticket purchase
	    - Individual ticket pdf - to all attendees (except purchaser)
	    - Order summary and a single pdf with all tickets - to the ticket purchaser
	- [x] Ticket cancellation - To the ticket purchaser
	- [x] Email verification email - To the user who registered
	- [x] A welcome email - After registration. Before email verification. - To the user who registered
	- [x] Forgot password reset email - To the user who registered
   	- [x] Email change notification (#4210) - To both the old email address and the new email address
   	- [x] Password change email - To the user
   	- [x] Event Role creation - To the invited user's email
	- [x] Event import success (with link to imported event) /failure - To the user (To be done after #3737)
	- [x] Event export success (with the exported file attached) /failure - To the user (To be done after #3731 )
	- [ ] Speakers call invite email - To the user who is being invited
	- [x] After event thank you email - To all registrered attendees & speakers
- Elasticsearch base for event searching (Low priority. Could be moved to next milestone depending on how the email notifications go. Not to be taken up until we have a PR for all of the above.) (#3467)

@fossasia/open-event-devs

**Comment by poush** [comment]
I'm thinking of one scenario here, i.e., if email changes, we have to check what is changed in after_create_object to see if email is changed and send them email only if email is changed. But what if we put this responsibility in email setter of User model. I guess it is not the responsibility of API to send the email.

**Comment by SaptakS** [comment]
@niranjan94 the remaining tasks are being followed in #3467  and #4225 . Closing this.
